Viktoria andreeva

diviana opened the pull request.

zzeni commented:
Ок са домашните. Определено си се постарал :)
На това с балсамика малко си направил desktop layout за мобилната версия, но това ще го изчистим като стигнем до респонсив дизайна.
На 4-то трябва да се оправят колоните, както си говорихме вчера.
Смятам че с display flex ще станат най-добре

diviana committed:
domashni ot lekciq 2

diviana committed:
empty html page responsive

Това е първото упражнение от лекция 6, празен html със зададени параметри за различни устройства.

zzeni commented on Lesson 6/empty.css:
```diff
@@ -0,0 +1,12 @@
+
```
моля те премести всички задачи в твоята папка
